foundation-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -526,12 +494,289 @@ ACPCore.TrackState("state", data);
 {% endtab %}
 {% endtabs %}
 
-## Collect PII
+## Set the advertising ID
```

Comment:
https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/identity/identity-api-reference#setadvertisingidentifier
We'd better move the `Identity` related API docs to the "identity-api-reference.md"